DocumentService/Controllers/DocumentsController.cs

```diff
@@ -107,9 +107,28 @@ public IActionResult GetAllSpecifiedDocuments(string idString)
         [ProducesResponseType(StatusCodes.Status200OK)]
         [ProducesResponseType(StatusCodes.Status400BadRequest)]
         [ProducesResponseType(StatusCodes.Status401Unauthorized)]
-        public IActionResult UpdateMetadataForDocument(int correlationId, string userName, string fileName, string fileContentType, string shortDescription, string submissionMethod, string fileLanguage, string documentTypes, string customMetadata)
+        public IActionResult UpdateMetadataForDocument(Guid documentId, string userName, string fileName, string fileContentType, string shortDescription, string submissionMethod, string fileLanguage, string documentTypes)
         {
-            return Ok();
+            var documentInfo = new DocumentInfo()
+            {
+                DocumentId = documentId,
+                UserCreatedById = userName,
+                DateLastUpdated = DateTime.UtcNow,
+                FileName = fileName,
+                Description = shortDescription,
+                SubmissionMethod = submissionMethod,
+                Language = fileLanguage,
+            };
+
+            try
+            {
+                var isUpdated = this.documentRepository.Update(documentInfo).Result;
+                return new JsonResult(isUpdated);
```

Comment:
Wouldn't return Ok(new { isUpdated }) be better

Reply:
JsonResult does the same thing as OK, but it formats the response as a JSONObject.

Reply:
Fair enough